README.md

```diff
@@ -1,206 +1,6 @@
-# Microsoft Power Platform Connectors
+Prerequisites
```

Comment:
@jforscht please follow our template for the readme file. Please include all the required sections. Please see an example [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) and for raw view for code [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md))